student_setup.py

```diff
@@ -0,0 +1,16 @@
+import sqlite3
+conn= sqlite3.connect("my-app.db")
```

Comment:
Perhaps in `db.py` it would be best to create a `DB_FILE` or `DB_NAME` constant that is used everywhere to access the database. Or maybe just create a conf or util.py. Up to your judgement